apps/marketplace/main.scss

```diff
@@ -334,3 +334,23 @@ table.defaultStyle {
   }
 }
 
+.noticeBar {
+  margin-top: 1.5rem;
+  margin-bottom: 1.5rem;
+  border-left: 10px solid #00698F;
+  background-color: #daf3fe;
+  color: #00698F;
+}
+
+.additionalInformationContainer {
```

Comment:
this style shouldn't be here.. also you should be able to use the generic styles

Reply:
resolved